### ISSUE TITLE
if condition?

### ISSUE THREAD

**Issue by paOol** [johntitus/node-horseman#194]
I am logging into instagram's api and on the first login, you are asked to authorize certain permissions.

how do I do a check or a condition based on that first login?

or even checking for http code could work too.

```javascript
  .type('input[name="username"]', userName)
  .keyboardEvent('keypress', 16777217) //tab
  .type('input[name="password"]', password)
  .click('input.button-green')

  .waitForNextPage()
```

for example, on first login there will be a $('input.confirm.button-green') to click to authorize.

so if i do .click, it'll work.

however, the on the second login, there is no need to authorize so the confirm.button-green is no longer there, so my code crashes because it's doing .click on a non existing item.

I looked at the methods and couldn't find one suitable for my case.
basically looking for something like

```javascript
if ( element.exists() ) 
// code
else{
//code
}
```

or 

```javascript
if(httpcode = 404){
//code
else{
//code
}
```

**Comment by awlayton** [comment]
There *is* such a method, and it's basically exacly what you called it in your example (`.exists()`).

However, you can't do it synchronously like that. Horseman is Promise based.

```javascript
  .type('input[name="username"]', userName)
  .keyboardEvent('keypress', 16777217) //tab
  .type('input[name="password"]', password)
  .exists('input.button-green')
  .then(function(exists) {
      if (exists) {
         return this.click('input.button-green')
      }
   })
  .waitForNextPage()
```

This is also `.status()` to get the HTTP response code.

Please ***read*** the Readme *thoroughly* before opening issues.
